src/main/java/core/basesyntax/serviceimpl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void write(String finalReport, File file) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file))) {
+            bufferedWriter.write(finalReport);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file");
+        }
+    }
+}
```

Comment:
The class FileWriterServiceImpl is not placed in the correct package according to the project structure requirements. It should be placed in a `service.impl` package.